.github/workflows/actions-pipeline.yml

```diff
@@ -0,0 +1,75 @@
+name: Build
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: false
+  
+on:
+  push:
+    branches: 
+      - 'main'
+      - 'release/*'
+  pull_request:
+    branches: 
+      - 'main'
+      - 'release/*'
+      - '!/docs/'
+      - '!/*.md'
+
+env:
+  RunPeVerify: true  
+  
+jobs:
+
+  build:
+
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+        projectloadstyle: [ net35, net40, net46, net6_0, net7_0 ]
+
+    runs-on: windows-latest  # For a list of available runner types, refer to
+                             # https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on
+
+    env:
+      Solution_Name: Open-XML-SDK.sln                         # Replace with your solution name, i.e. MyWpfApp.sln.
```

Comment:
can you remove these comments that aren't really necessary once we get it set up?